frontend/src/components/pages/PlayPage.js

```diff
@@ -57,15 +61,9 @@ export default function PlayPage() {
       .reduce((sum, currentBet) => sum + currentBet);
```

Comment:
Maybe do the maping in the reducer 😆 ?

Reply:
you mean like this? 
```
return game.betSession.players.reduce(
      (sum, player) => sum + player.currentBet
    );
```
it does not work, becaus sum is an object... Or do you have an Idea, to work around this?

Reply:
reduce takes a second parameter, the initial value 
```
game.betSession.players.reduce((sum, player) => sum + player.currentBet, 0);
```